src/printer.js

```diff
@@ -1074,7 +1074,7 @@ function genericPrintNoParens(path, options, print) {
               n.selfClosing ? line : softline
             ])
           ),
-          n.selfClosing ? "/>" : ">"
+          n.selfClosing ? "/>" : ">",
```

Comment:
Currently we don't use trailing commas. Although this will be stripped out when we reprint the file so doesn't really matter :)